config/initializers/ruby_money.rb

```diff
@@ -1,8 +1,16 @@
 # frozen_string_literal: true
 
+require 'money/bank/currencylayer_bank'
+
 Money.locale_backend = :i18n
 
-eu_bank = EuCentralBank.new
-Money.default_bank = eu_bank
+mclb = Money::Bank::CurrencylayerBank.new
+mclb.access_key = ENV.fetch("CURRENCY_LAYER_API_KEY", "")
+mclb.currencylayer = true
+mclb.ttl_in_seconds = 86_400
+mclb.cache = 'rails_money_cache.txt' # To be replaced with proper cache path.
```

Comment:
Does the gem only support file caching? Would love to use Redis for this instead

Reply:
Okay I didn't knew we were having redis. I couldn't get to see anywhere officially how to use it.

I tried `mclb.cache = :redis_cache_store, { url: EnvConfig.CACHE_REDIS_URL }` and I am not sure how to test if it worked.